pkg/cli/cmds/agent.go

```diff
@@ -155,6 +156,14 @@ var (
 		Destination: &AgentConfig.EnableSELinux,
 		EnvVar:      version.ProgramUpper + "_SELINUX",
 	}
+	LBServerPort = cli.IntFlag{
```

Comment:
What is this needed for?

Reply:
this is interesting workaround that I added, basically this sets the load balancer port instead of using a random port, I defaulted this value if not set to 6666 and this way it gets assigned to APIServerPort in server config and all kubeconfigs gets generated with this endpoint 127.0.0.1:6666 and also this gets assigned to the loadbalancer for api servers

Reply:
As @erikwilson suggested in Slack, can we make this the same as the apiserver client port? It's only used on nodes without a local etcd server, so the port should be available.